src/index.css

```
@@ -5,6 +5,8 @@
  */
 
 @import 'tailwindcss';
+@config "../tailwind.config.js";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@config`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)